applications/sdp/mspi/sample.yaml

```diff
@@ -13,3 +13,5 @@ tests:
       - ci_build
       - sysbuild
       - mspi
+    extra_args:
+      - CONFIG_SDP_DEVELOPER_MODE=y # temporary fix for CI
```

Comment:
Please, remove this line, fix for ASM developer mode was merged already, so CI should pass in this PR without this change.